cms/envs/aws.py

```diff
@@ -283,6 +283,10 @@
 DATABASES = AUTH_TOKENS['DATABASES']
 
 MODULESTORE = convert_module_store_setting_if_needed(AUTH_TOKENS.get('MODULESTORE', MODULESTORE))
+
+# Iterable containing dotted paths to modulestore wrappers
+MODULESTORE_WRAPPERS = ENV_TOKENS.get('MODULESTORE_WRAPPERS', ())
```

Comment:
@cpennington updated. I've touched up each of the settings modules a bit so they follow this pattern.